en/docs/what-is-devant.md

```diff
@@ -1,18 +1,20 @@
 # What is Devant?
 
-Devant allows you to manage your own AI integration agents. Integrate any AI agent into the integrations you build in low-code and pro-code, and jump from siloed systems to intelligent digital experiences with Devant by WSO2—the AI iPaaS that your AI Agents can call “home”.
+Devant is a powerful IPaaS with first-class AI support. Integrate any AI agent into the integrations you build in low-code and pro-code, and move away siloed systems to intelligent digital experiences with Devant by WSO2—the AI iPaaS that your AI Agents can call “home”.
```

Comment:
This is my bad, it should be "move away from".

Reply:
Updated